### DISCUSSION TITLE
Active Users in Organization

### DISCUSSION THREAD

**Discussion by abjinugurxo** [community/community#87705] [Enterprise]
Can I list all the active users in my Organization?
Active users = Users with at least one code commit in the last 1 month.
I checked REST API and GraphQL , but didn't find any solution.

**Comment by murat-yasar** [comment]
You can use a search query to enter in the search bar on your GitHub page. By using the following query in the search bar, you can have the list of the commits since the given date (in the following example: since 31.12.2024):

org:ORGANIZATION_NAME type:commits committer-date:>2023-12-31

And then you have some options at the filter on the left (like commits, users, repos etc) and the sort filter on top right corner of the result list (like recently committed, etc). 

You can save the query if you want by clicking "save" icon next to the sort filter. And the next time, you can just change the committer-date to see the latest results.

Maybe it's not the exact thing you are looking for but it's sure, it helps.

**Comment by mecodeatlas** [comment]
Hey @abjinugurxo 👋
We’re going through some older unanswered discussions and I know it’s been a while, but I wanted to check in and see if you were able to get this sorted out.

If @murat-yasar’s reply helped you solve the issue, could you mark their response as the answer?
Marking an answer highlights the comment and its replies, making it easier for future visitors to quickly find what worked.

![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

Thank you very much!
